diff --git a/node_modules/@vangbanlanhat/fca-unofficial/src/actions/listenMqtt.js b/node_modules/@vangbanlanhat/fca-unofficial/src/actions/listenMqtt.js
index c24138d..4eecaad 100644
--- a/node_modules/@vangbanlanhat/fca-unofficial/src/actions/listenMqtt.js
+++ b/node_modules/@vangbanlanhat/fca-unofficial/src/actions/listenMqtt.js
@@ -177,6 +177,7 @@ function listenMqtt(defaultFuncs, api, ctx, globalCallback) {
 	});
 
 	mqttClient.on('message', function (topic, message, _packet) {
+			console.log(`[RAW MQTT] Topic: ${topic}, Payload: ${message.toString().slice(0, 300)}`);
 			var jsonMessage;
 			try {
 				jsonMessage = JSON.parse(message.toString());
diff --git a/node_modules/@vangbanlanhat/fca-unofficial/src/index.js b/node_modules/@vangbanlanhat/fca-unofficial/src/index.js
index f1dc174..4d3eb9c 100644
--- a/node_modules/@vangbanlanhat/fca-unofficial/src/index.js
+++ b/node_modules/@vangbanlanhat/fca-unofficial/src/index.js
@@ -192,6 +192,7 @@ function buildAPI(globalOptions, html, jar) {
         log.info("login", `[Unused] Polling endpoint: ${legacyFBMQTTMatch[6]}`);
       } else {
         log.warn("login", "Cannot get MQTT region & sequence ID.");
+        require('fs').writeFileSync('D:\\code\\sao-sang-bridge\\facebook_home.html', html, 'utf8');
         noMqttData = html;
       }
     }
